tests/unit.cpp

```diff
@@ -951,7 +951,6 @@ TEST(db, docs) {
 
     // BSON
     bson_error_t error;
-
```

Comment:
Let's remove such things from commits. Otherwise, it is hard to navigate any significant changes in file history.